src/components/partitions/PartitionTable.vue

```diff
@@ -46,7 +46,11 @@ const navigateToPartition = async (partitionId: string) => {
         <td class="px-1 py-4 pl-6">
           {{ partition.name }}
         </td>
-        <td class="px-1 py-4 pr-6">{{ partition.budget }}円</td>
+        <td class="px-1 py-4 pl-6">
+          <div v-if="partition.budget !== null">{{ partition.budget }} 円</div>
+          <div v-else>指定なし</div>
```

Comment:
こういうのもあります
この処理関数かコンポーネントに切り出しちゃってもいいかもね
```suggestion
          <div>
            {{
              partition.budget === null ? '指定なし' : partition.budget + '円'
            }}
          </div>
```